ephios/core/forms/users.py

```diff
@@ -464,3 +464,9 @@ def __init__(self, *args, **kwargs):
 
     def update_preferences(self):
         self.user.preferences["notifications__notifications"] = self.cleaned_data
+
+
+class UserOwnDataForm(ModelForm):
+    class Meta:
+        model = UserProfile
+        fields = ["language"]
```

Comment:
I'd prefer if the label would be "Interface Language" as it does not change the language of user provided values (e.g. event description).

Reply:
is now preferred language